dex/testing/loadbot/loadbot.go

```diff
@@ -352,6 +354,13 @@ func run() error {
 	flag.BoolVar(&trace, "trace", false, "use trace logging")
 	flag.IntVar(&m, "m", 0, "for compound and sidestacker, m is the number of makers to stack before placing takers")
 	flag.IntVar(&n, "n", 0, "for compound and sidestacker, n is the number of orders to place per epoch (default 3)")
+	flag.Int64Var(&rateIncrease, "increase", 0, "for compound and sidestacker, increase is applied to every order and increases or decreases price by the chosen flat rate, use to create a market trending in one direction. Must be a multiple of the market's rate step (default 0)")
```

Comment:
For `rateIncrease`, how about doing like with `oscStep` and having it be a number of rate steps.  It looks like it can be multipled by rate step everywhere it's used.
Also, since it can be negative (right?) maybe `rateShift` / `"shift"` would be more clear?

Reply:
change the flag `rateIncrese` -> `rateShift`